src/DoctrineDataFixtureModule/Command/ImportCommand.php

```diff
@@ -29,34 +29,50 @@
 use Doctrine\Common\DataFixtures\Executor\ORMExecutor;
 use Doctrine\Common\DataFixtures\Purger\ORMPurger;
 use DoctrineDataFixtureModule\Loader\ServiceLocatorAwareLoader;
-use Zend\ServiceManager\ServiceLocatorInterface;
+use Doctrine\ORM\EntityManagerInterface;
 
 /**
- * Command for generate migration classes by comparing your current database schema
- * to your mapping information.
+ * Command to import Fixtures
  *
- * @license http://www.opensource.org/licenses/lgpl-license.php LGPL
```

Comment:
We have the license already set as MIT for the library. Unfortunately it is set differetnly in the files.